docs/management_api.yml

```diff
@@ -923,16 +1003,63 @@ definitions:
       limit:
         type: integer
         description: |
-            Storage limit in bytes. If set to 0 - there is no limit for storage.
+            storage limit in bytes. if set to 0 - there is no limit for storage.
       usage:
         type: integer
         description: |
-            Current storage usage in bytes.
+            current storage usage in bytes.
     required:
       - limit
       - usage
     example:
       application/json:
         limit: 1073741824
         usage: 536870912
-
+  Release:
+    description: Groups artifacts with the same release name into a single resource.
```

Comment:
Grouping more of conternal implementation, conceptually this is a stand alone resource.

Reply:
so..."Software release definition"?

Reply:
Maybe something along the lines:

Release is a group of mender artifacts containing same version of software that targets variety of different platform types.

Reply:
fixed, but used `...different device types` instead of '...platform types' to be 1:1 with field names